kitchen.yml

```diff
@@ -0,0 +1,13 @@
+---
+verifier:
+  name: inspec
+provisioner:
+  name: chef_solo
+  # enforce_idempotency: true # 'delete.com' test site gets created & deleted each converge
```

Comment:
Let's go ahead and fix the test resource so that doesn't use the same resource to test this causing the problem. I'd rather we have this enabled if possible.